src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPresenter.java

```diff
@@ -509,6 +525,33 @@ public void onResponseReceived(String handle)
          }
       });
 
+      dialog.getDirectoryChooser().addValueChangeHandler(new ValueChangeHandler<String>() {
```

Comment:
Maybe we should only add this value change handler when `session_.getSessionInfo().isVcsAvailable` is true? (otherwise there's no point as we'd never show git options)

Reply:
done 👍